src/lib/config/polygon.json

```diff
@@ -6,9 +6,9 @@
   "shortName": "Polygon",
   "network": "polygon",
   "unknown": false,
-  "rpc": "https://polygon-mainnet.infura.io/v3/daaa68ec242643719749dd1caba2fc66",
-  "ws": "wss://polygon-mainnet.g.alchemy.com/v2/ODJ9G5Ipv-Hb2zTWMNbUFIqv9WtqBOc2",
-  "loggingRpc": "https://polygon-mainnet.g.alchemy.com/v2/hsAEQ4l4c6uz3ciAdQ9w3ODWjVcND-Ti",
```

Comment:
Anyone know why we have two different alchemy keys for Polygon? Which one is correct?